LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 Crown Copyright (Ministry of Justice)
+Copyright (c) 2023 Crown Copyright (Ministry of Justice)
```

Comment:
I was told we should update the year like 2022 - 2023.

Reply:
All other examples I can find are single years, but I'm curious.

Reply:
Sure [tech guidance](https://technical-guidance.service.justice.gov.uk/documentation/standards/licencing-software-or-code.html#use-the-correct-copyright-notice)